scripts/dev_env/4_database_postgresql.sh

```diff
@@ -0,0 +1,104 @@
+# Ask and install Navitia databases (PostgreSQL format)
+ask_db_install() {
+    if [ -z `command -v psql` ]; then
+        print_error "\"psql\" is not available, script is aborting"
+        exit
+    fi
+
+    if [ -z `command -v virtualenv` ]; then
+        print_error "\"virtualenv\" is not available, script is aborting"
+        exit
+    fi
+
+    print_question "Do you wish to configure Navitia databases (PostgreSQL format) ? [Y/n]"
+    user_answer
+    if [ $? == 1 ]; then
+        print_info "Configuring Navitia databases ..."
+        db_create
+    else
+        print_info "Navitia databases is not configured"
+    fi
+}
+
+# Create Navitia databases (PostgreSQL format)
+db_create() {
+    db_create_navitia
+    db_create_jormungandr
+}
+
+# Create Navitia database
+db_create_navitia() {
+    local db_user_password
+    db_user_password='navitia'
```

Comment:
Can you inform the user with a log, that he set up database with this kind of values (password, user, name) ?

Reply:
Good idea, I will dot it.
Thx